### PR TITLE
fix: remove proteins_pytorch_geometric from nightlies.

### PR DESCRIPTION
**Pull request by ioga**
## Description

Seems like `proteins_pytorch_geometric` example sometimes has convergence issues which breaks nightlies. Remove it for the nightlies for now, pending an investigation. Also set batch_size to 20 to align with the original repo example.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: e2e_tests/tests/nightly/test_capability.py [hunk 1]
```diff
@@ -82,13 +82,3 @@ def test_word_language_lstm_const() -> None:
     config["hyperparameters"]["tied"] = False
 
     exp.run_basic_test_with_temp_config(config, conf.nlp_examples_path("word_language_model"), 1)
-
```
Comment (RehanSD) [inline e2e_tests/tests/nightly/test_capability.py:85]: Should we comment this out instead of just deleting so we know to return to it eventually?

**Review by RehanSD — COMMENTED** [review]
LGTM except for the one comment I left. Once we address that I'm happy to merge!

File: e2e_tests/tests/nightly/test_capability.py (hunk 1 above)
Comment (ioga) [reply e2e_tests/tests/nightly/test_capability.py:85]: I think this is what git history is for. you don't want a lot of dead commented code laying around.

**Review by RehanSD — APPROVED** [review]
LGTM!

**Pull request merged**